김현승/user-api.v2.yaml

```
@@ -0,0 +1,339 @@
+openapi: 3.0.3
+info:
+  title: Compass User Domain API
+  description: |
+    # User 도메인 API 명세서 (v2.0)
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)